CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [11.0.0](https://github.com/CrowdStrike/eslint-config-crowdstrike/compare/v10.1.1...v11.0.0) (2024-07-16)
```

Comment:
This is auto-generated. You shouldn't need to touch this file.

Reply:
ah, so `yarn run release` gets run after i push up?

Reply:
yep